Doc/library/fractions.rst

```diff
@@ -25,8 +25,8 @@ another rational number, or from a string.
 
    The first version requires that *numerator* and *denominator* are instances
    of :class:`numbers.Rational` and returns a new :class:`Fraction` instance
-   with value ``numerator/denominator``. If *denominator* is ``0``, it
-   raises a :exc:`ZeroDivisionError`.
+   with value equal to ``numerator/denominator`` where the denominator is positive.
```

Comment:
Sorry, I don't think it's an improvement.

Reply:
I agree with @skirpichev. We can leave "equals", but "where the denominator is positive" is incorrect or at least ambiguous. "Denominator" refers to the argument of the `Fraction` constructor, not the `denominator` property.

Reply:
I also agree. It's as if we require *denomintor* to be positive which is not the case. So I suggest to revert this PR.

Reply:
I opened pr to fix docstrings in Rational class: https://github.com/python/cpython/pull/136800  It's a accompanying pr to #122464, which fix sphinx docs (and stuck in review).

But maybe it's ok to combine both and include reversion of this pr?  All that only affects docs and IMO - fine for backporting up to 3.13.

What do you think?

Reply:
@serhiy-storchaka @skirpichev we discussed this at the sprint and decided that `*denominator*` (italics) referred to the argument, but `denominator` (normal/roman) in the context of the sentence referred to the new argument. Regardless, it simplifies the arguments, so it is not always the `denominator` argument that is set as the class attribute.

A

Reply:
It says: "a new :class:`Fraction` instance with value equal to ``numerator/denominator``". "denominator" refers to argument, not to the property. Otherwise this sentence does not make sense. Then you say that in "where the denominator is positive" "denominator" refers to something other. But it clearly refers to the same thing as in previous expression. Your interpretation requires interpreting the same word in two different ways simultaneously. Even if it makes sense, it is very unclear and confusing. And to me, it just looks wrong.